### DISCUSSION TITLE
Make media query `mq` utility available with useStyles hook

### DISCUSSION THREAD

**Discussion by kutsan** [jpudysz/react-native-unistyles#266] [Feature Requests]
It would be great to have the `mq` function or a similar utility available within the `useStyles` hook. My use case involves conditionally rendering components based on the current screen size. Currently, `mq` returns a symbol, making it unusable within components.

```typescript
const { styles, mq } = useStyles(styleSheet)

const isLargeScreen: boolean = mq.only.width('lg') // or something like that.
```

**Comment by jpudysz** [comment]
Hey 👋 I built my own custom hook for that. I agree that it will be cool to expose something like that from Unistyles. 

Currently I'm working on v.3.0 and by default Unistyles won't be able to re-render your components. I will create a dedicated hook for devs who want to subscribe for changes. Such hook could be customized to listen for breakpoint changes. I will think about it!

**Reply by kutsan** [reply]
Yes, we're using a custom hook as well, but since we have access to breakpoints, I thought it would be cool to have something similar to [`matchMedia`](https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia) for the web.

But that sounds great! Really looking forward to v3. Thanks a lot!
